configs/local/docker-compose.yml

```diff
@@ -1,88 +1,84 @@
 version: "3"
 
 services:
-    web:
-        image: docker.cloudsmith.io/convoy/convoy/frain-dev/convoy:v23.9.2
-        command: ["/start.sh"]
+    server-1:
+        image: rtukpe/bench:v2
+        command: ["./cmd", "server", "--config", "convoy.json", "--port", "3000"]
         volumes:
             - ./convoy.json:/convoy.json
         restart: on-failure
+        networks:
+            backendCluster:
+                ipv4_address: 172.16.0.12
         ports:
-          - "5005:5005"
-        healthcheck:
-          test: ["CMD-SHELL", "wget -q --spider localhost:5005/health"]
-          interval: 5s
-          timeout: 15s
-          retries: 3
-          start_period: 5s
-        depends_on:
-            - postgres
-            - redis_server
-            - typesense
+            - "3000:3000"
+
+    server-2:
+        image: rtukpe/bench:v2
+        command: ["./cmd", "server", "--config", "convoy.json", "--port", "3001"]
+        volumes:
+            - ./convoy.json:/convoy.json
+        restart: on-failure
+        networks:
+            backendCluster:
+                ipv4_address: 172.16.0.11
+        ports:
+            - "3001:3001"
 
```

Comment:
Does this need to be here or a separate compose file?

Reply:
I'm reverting the compose file changes, I left it because I might still need to make changes

Reply:
Okay